WolvenKit.Modkit/RED4/Tools/MeshTools.cs

```diff
@@ -550,12 +557,26 @@ public static List<RawMeshContainer> ContainRawMesh(MemoryStream gfs, MeshesInfo
                     meshContainer.indices[i] = gbr.ReadUInt16();
                 }
 
+                if (meshname.Length>0)
+                {
+                    meshContainer.name = mergeMeshes ?
+                                        info.appearances.Keys.FirstOrDefault("default") :
+                                        Path.GetFileNameWithoutExtension(meshname)+"_submesh_" + Convert.ToString(index).PadLeft(2, '0') + "_LOD_" + info.LODLvl[index];
```

Comment:
Can you pull this in from the meshinfo? Or maybe throw the naming convention in a function? Might be over-defensive but could prevent accidentally changing one but not the other.

Reply:
The mesh info doesnt have the parent object name as part of it, thats part of the issue. Could change the meshinfo, but tbh it was gonna start effecting a lot more of the code and I was already borderline out of my depth. Am aware this only fixes the default export path at the minute, your experimental one goes a different route right?

Reply:
I don't even remember what it does tbh, but now that you mention it, it's probably better to try to make the change in a more holistic overhaul yeah.